### ISSUE TITLE
Linksharing / Hide the .file-placeholder in the UI

### ISSUE THREAD

**Issue by boshevski** [storj/storj#5563]
Creating new folder adds ".file_placeholder" object which we need to hide in the UI when browsing linksharing folders.

![Image](https://user-images.githubusercontent.com/3217669/217618469-6ada4cf8-c4b5-4355-97fd-14063945a9b7.png)

**Comment by jtolio** [comment]
i strongly disagree that the solution here is to change linksharing's folder listing behavior to not show files.

if anything, the fix is to make the UI not require creating .file_placeholder to begin with.

**Comment by boshevski** [comment]
I didn't say to change the listing behavior, the UI can just hide the file visually only, or like you said make the UI not require creating the placeholder file.

**Comment by jtolio** [comment]
okay, are you imagining that the frontend has a checkbox that says "show hidden files" that starts off disabled or something?

**Comment by boshevski** [comment]
No, just specifically hide this placeholder object only in the UI, since
there is no need for it from user's perspective. Same as we don't show it
in the file browser.

**Comment by pwilloughby** [comment]
Is there a reason we don't create an object with a trailing slash in the name instead?  "new-folder/.file_placeholder" -> "new-folder/"?  The Uplink cli, linksharing, gateway, aws cli, and the satellite object browser all seem to handle it like you would want and expect.
